src/Resources/ProductLists/ProductLists.ts

```diff
@@ -7,11 +7,8 @@ export default class ProductLists extends AbstractResource {
         this.defaultAccess = 'guest';
     }
 
-    public get: (
-        sharingSlug: string,
-        ...args: HttpClientConfig
-    ) => HttpClientResponse = this.createMethodFromPartialSpec({
+    public get: (slug: string, ...args: HttpClientConfig) => HttpClientResponse = this.createMethodFromPartialSpec({
         method: 'GET',
-        path: '/product-lists/{sharing-slug}',
```

Comment:
I believe this could be `sharingSlug` if you were fine with that btw

Reply:
Yeah can do - happy for me to change that on develop branch?